### ISSUE TITLE
视频播放界面支持全面屏手机么？ 竖屏隐藏导航栏后，播放视图被刘海遮挡，横屏界面也会被刘海遮挡

### ISSUE THREAD

**Issue by stoneUoU** [changsanjiang/SJVideoPlayer#483]
**你期望的结果是什么?**
像其它播放器一样支持刘海屏手机

**截图**
<img width="339" alt="WechatIMG145" src="https://user-images.githubusercontent.com/24381761/143974783-4b6113a2-a13c-432a-af66-d6f47da79b02.png">

<img width="1403" alt="WechatIMG146" src="https://user-images.githubusercontent.com/24381761/143974811-314742d3-f5c8-48a8-9524-2766a1e18938.png">

**额外的**
像优酷，尤其是横屏播放界面   适配的非常好

![WechatIMG67 1](https://user-images.githubusercontent.com/24381761/143975088-c64ba679-b36a-44da-932a-25276b5f94b5.jpeg)

![WechatIMG66](https://user-images.githubusercontent.com/24381761/143975198-b8ea2aaa-27f2-4b24-a74f-5f9fdb0962b7.jpeg)

**Comment by changsanjiang** [comment]
```oc
    [_videoPlayer.view mas_makeConstraints:^(MASConstraintMaker *make) {
        if (@available(iOS 11.0, *)) {
            make.top.equalTo(self.view.mas_safeAreaLayoutGuideTop);
        } else {
            make.top.offset(20);
        }
        make.left.right.offset(0);
        make.height.equalTo(self.view.mas_width).multipliedBy(9/16.0f);
    }];
```

**Comment by stoneUoU** [comment]
> [_videoPlayer.view mas_makeConstraints:^(MASConstraintMaker *make) {
>         if (@available(iOS 11.0, *)) {
>             make.top.equalTo(self.view.mas_safeAreaLayoutGuideTop);
>         } else {
>             make.top.offset(20);
>         }
>         make.left.right.offset(0);
>         make.height.equalTo(self.view.mas_width).multipliedBy(9/16.0f);
>     }];

这个感觉不是我要的效果，哈哈
